surveys/2021-contributor-survey/questions.md

```diff
@@ -0,0 +1,495 @@
+# Survey questions
+
+Whether or not you contribute to the Rust Project <https://github.com/rust-lang>
+today, we want to hear from you!
+
+The Rust Foundation has created this survey to help us identify the main
+problems negatively affecting active, past, and potential contributors to the
+Rust Project to help us plan programs that can most effectively support the
+contributors and maintainers of the Rust Project.
+
+Unless you choose to enter your email, your answers will be anonymous. Any
+personal data you submit as a part of this survey will be handled in accordance
+with our policy as described in our Frequently Asked Questions:
+
+https://github.com/rust-community/team/wiki/State-of-the-Rust-Language-Community-Survey-FAQ
+
+We estimate it will take about ? minutes to complete.
+
+> TODOs:
+>
+> - Evaluate survey length (the interviews took ~20-30 minutes)
+> - Can the community survey FAQ be generalized for all surveys such as this or
+>   the trademark surveys?
+> - Resolve and close open issues related to this survey on `rust-lang/surveys`
+> - Are these questions being presented in the optimal order?
+> - Make sure none of the major cohort sections are leaving out relevant
+>   questions that are asked of the other major cohorts.
+
+## Collaboration Interview
+
+### Have you contributed or ever considered contributing to the official Rust project (e.g. any repo that is part of the rust-lang Github org)?
+
+- I am an active contributor to the Rust Project. [NEXT](#for-active-contributors)
+- I was an active contributor to the Rust Project but am not any longer. [NEXT](#for-inactive-contributors)
+- I have tried contributing to the Rust Project but stopped before completing any contributions. [NEXT](#for-attempted-contributors)
+- I have wanted to contribute to the Rust Project but have not been able to figure out how. [NEXT](#for-potential-contributors)
+- I have never wanted to contribute to the Rust Project. [NEXT](#for-non-contributors)
+
+> **justification**
+>
+> Fundamental for cohort analysis.
+
+## For active contributors
+
+### How did you get involved in the Rust Project?
+
+Select all that apply:
+
+- I wanted to improve my Rust skills and learn from the experts maintaining Rust.
+- I wanted to implement features that I needed as a user of Rust.
+- I wanted to give back to the Rust community.
+- I wanted to add to my portfolio of open source work for my resume.
+- I was encouraged to contribute by an existing Rust contributor / maintainer.
+- My work assigned open source contributions to me as part of my job.
+- Other: Free form
+
+> **justification**
+>
+> Useful in identifying paths into the project that are successful for future
+> contributors to follow.
+
+### How long have you been contributing to the Rust Project?
```

Comment:
Are we leaving room for people who have contributed, stopped, and began contributing again? Is this potentially a large enough group for us to care about?

Reply:
> Are we leaving room for people who have contributed, stopped, and began contributing again?

We could definitely do so. This may be slightly covered by

> Is this potentially a large enough group for us to care about?

My intuition is that the group size doesn't matter. I don't expect people who have left and come back to have substantially different issues, though if you have a different intuition I'd be interested to hear your perspective.